### PR TITLE
Movies API

### PR DESCRIPTION
**Pull request by storres20**
# In this milestone we did the following activities:
- Add "moviesapi" branch
- Set up react app
- Set up Redux store and Reducers
- Set up linters
- Install React Redux, React Router, React Bootstrap
- Add Home, Details, About pages
- Add Searchbar, SearchCat, Navbar and other components
- Add testing files

### CODE REVIEW

File: src/pages/HomePage/HomePage.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <ul className={styles.moviesGrid}>
+        {categories.map((movie) => (
+          <Link key={movie.id} to={`/detailPage/${movie.id}`}>
+            <li className={styles.movieCard}>
+              <div>{movie.name}</div>
+            </li>
+          </Link>
+        ))}
+      </ul>
+
```
Comment (omar-muhamad) [inline src/pages/HomePage/HomePage.jsx]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Layout%3A%20composition%20and%20space%20between%20elements.), For the home page, Please make the movies categories shaped in a two-column grid and the background color is staggered like a chessboard to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

### ⚠️This is a mobile application, So please focus on the mobile view only and make sure that you follow the given design⚠️

File: src/pages/DetailPage/MoviesGrid.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+  return (
+    <div>
+      {loading
+        ? (
+          <div>
+            <div className={styles.flexTitle}>
+              {(category.length === 0) ? '' : <h2 className={styles.title}>{category[0].name}</h2>}
+              <Link to="/"><div className={styles.button}>Back</div></Link>
+            </div>
```
Comment (omar-muhamad) [inline src/pages/DetailPage/MoviesGrid.jsx]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Layout%3A%20composition%20and%20space%20between%20elements.), For the details page, please make sure that the back button appears on the left side of the navbar to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: src/tests/AboutPage.test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import AboutPage from '../pages/AboutPage/AboutPage';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <AboutPage />,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (omar-muhamad) [inline src/tests/AboutPage.test.js:12]: You only add screenshot tests to your app:

- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20unit%20tests%20for%20pure%20functions%20in%20your%20Redux%20code.), Please add unit tests for pure functions in your Redux code.
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library), Please add integration tests for your application using the React Testing Library like mocking the connection to the Redux Store.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @storres20,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by franclobo — APPROVED** [review]
Hi @storres20, 

# STATUS: APPROVED 😃 

gif-good-job-79

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

## Highlights

- No linter errors ✔️ 
- Correct Git flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**